wiki/FitNesseRoot/HsacExamples/SlimTests/HttpTests.wiki

```diff
@@ -101,17 +101,32 @@ The commands/keywords (i.e. public methods) added by !-nl.hsac.fitnesse.fixture.
+|all XPath matches <expression>                                                            |Evaluates the expression against the last response received and returns all results. Intended to be used with 'show' command.                                                                        |
+|register prefix <prefix> for namespace <url>                                              |Defines a prefix to be used to identify a namespace in subsequent 'xPath' evaluation.                                                                                                                |
+|xPath int <expression>                                                                    |Evaluates the expression against the last response received, returning a whole number (allowing [[relative value tests][.FitNesse.UserGuide.WritingAcceptanceTests.SliM.ValueComparisons]]).         |
+|xPath double <expression>                                                                 |Evaluates the expression against the last response received, returning a floating point number (allowing [[relative value tests][.FitNesse.UserGuide.WritingAcceptanceTests.SliM.ValueComparisons]]).|
+|raw xPath <expression>                                                                    |Evaluates the expression against the last response received, ignoring any (SOAP) faults received (which cause 'xPath <expression>' to throw an exception).                                           |
+|create file <fileName> from base64 content of <expression>                                |Evaluates the expression against the last response received, expecting the found element to contain a base64 encoded value. That value will be saved to a file and a link to the file returned.      |
+|repeat until xPath <expression> is <value>                                                |If 'expression' does not evaluates to 'value' for the last response: repeats last request until it does. Do not forget to set 'repeat at most times', or else the loop may run endlessly!            |
+|validate response against xsd <XSD Schema>                                                |Validates the response xml against the given XSD Schema                                                                                                                                              |
+|validate response against xsd file <XSD File>                                             |Validates the response xml against the XSD Schema in the given XSD File                                                                                                                              |
+|'''WS Security configuration options'''                                                                                                                                                                                                                                                         |
```

Comment:
This sentence seems incomplete

Reply:
It was meant as a sub-header to visually separate the ws sec methods in the documentation. Renders like this: 
![image](https://user-images.githubusercontent.com/2232710/75230031-63011800-57b3-11ea-8fd3-6f57d2559bf5.png)

Reply:
A separation is nice indeed, that's why I thought a separate subclass, and table here, would be better.

Reply:
Agree, separated in WsSecurityHttpTest now